pkg/sql/logictest/testdata/logic_test/pg_catalog

```diff
@@ -1358,14 +1358,14 @@ ORDER BY oid
 oid     typname        typndims  typcollation  typdefaultbin  typdefault  typacl
 16      bool           0         0             NULL           NULL        NULL
 17      bytea          0         0             NULL           NULL        NULL
-18      char           0         3903121477    NULL           NULL        NULL
-19      name           0         3903121477    NULL           NULL        NULL
+18      char           0         3403232968    NULL           NULL        NULL
```

Comment:
seems like `char` in PG has a typcollation of 0

Reply:
hmm, i think this is a separate issue in that this is what pg_type reports, which is different.
i think that's a separate issue. we currently base this column on the collation of the type...